firmware/user/modes/mode_tunernome.c

```diff
@@ -358,6 +428,16 @@ static void ICACHE_FLASH_ATTR tunernomeUpdate(void* arg __attribute__((unused)))
             // Don't do anything when paused
             if(tunernome->pause)
             {
+                plotRect((OLED_WIDTH - PAUSE_SPACE_WIDTH) / 2 - PAUSE_WIDTH,
```

Comment:
Love this pause

Reply:
I just changed it to be solid rectangles. What's your preference?

Reply:
~ ~ ~ Follow your heart ~ ~ ~

Reply:
Going with plotRect I think.